ios/PurchasesHybridCommon/PurchasesHybridCommon/RCPurchases+HybridAdditions.h

```diff
@@ -18,14 +18,16 @@ NS_ASSUME_NONNULL_BEGIN
                        observerMode:(BOOL)observerMode
               userDefaultsSuiteName:(nullable NSString *)userDefaultsSuiteName
                      platformFlavor:(nullable NSString *)platformFlavor
-              platformFlavorVersion:(nullable NSString *)platformFlavorVersion;
+              platformFlavorVersion:(nullable NSString *)platformFlavorVersion
+                  dangerousSettings:(nullable RCDangerousSettings *)dangerousSettings;
```

Comment:
this would technically make it a major release

Reply:
That makes sense!